_lab/lab05.md

```diff
@@ -0,0 +1,200 @@
+
+A reminder about working with a pair programming partner:
+
+* Your partner must be enrolled in the same lab section as you.
+* You and your partner must agree to work together outside of lab section in case you do not finish the lab during your lab time. You must agree to reserve at least two hours outside of lab section to work together if needed (preferably during an open lab hour where you can work in Phelps 3525 and ask a mentor for help). You are responsible for exchanging contact information in case you need to reach your partner.
+* If you choose to work with a partner, then you must choose a partner you have not worked with before.
+* You MUST add your partner on Gradescope when submitting your work <strong>*<u>EACH TIME</u>*</strong> you submit a file(s). After uploading your file(s) on Gradescope, there is a "Group Members" link at the bottom (or "Add Group Member" under "Groups") where you can select the partner you are working with. Whoever uploaded the submission must make sure your partner is part of your Group. Click on "Group Members" -> "Add Member" and select your partner from the list.
+
+Once you and your partner are in agreement, choose an initial driver and navigator, and have the driver log into their account.
+
+## Getting started
+
+* Step 1: Log on and open up a terminal window
+This is done following the steps you have performed in lab00.
+* Step 2: Create a directory in your cs8 directory named {{page.num}}.
+* Step 3: Start IDLE
+
+The terminal command for this is "`idle3 &`".  When you have the IDLE window up, you are ready for some programming!
+
+## What to program?
+
+In this lab assignment, you are going to make your own Scrabble word finder function, `scrabbleWords()`.  In the end, you will simply input a string of letters and the program will print out (to the screen and to a file) a list of all the possible words you can make along with their point values in descending order (neglecting things like triple letter, double word squares, etc. in the real game of Scrabble).  For example, if I input `'bouni'` as my string of letters, this is what I get:
+
+```
+>>> scrabbleWords('buoni')
+obi      5
+nub      5
+nob      5
+nib      5
+bun      5
+bio      5
+bin      5
+bi       4
+uni      3
+ion      3
+on       2
+nu       2
+no       2
+in       2
+u        1
+i        1
+```
+
+So, how did our program know which letter combinations were valid words?......We have to specify a file of words, which you can find here: [wordlist.txt](https://ucsb-cs8.github.io/f19/lab/lab05/wordlist.txt).
+
+This file must be downloaded (right click and "save as") and put into your `{{page.num}}` directory before you begin, so do that now.  Note that this file contains a fairly complete list of English words, so beware that there may be some expletive and/or raunchy words - please don't hold me personally responsible if you are offended.  Perhaps this will be motivation for some of you to complete the assignment.  
```

Comment:
Sounds like these words were actually removed? So we can take out this part of the instructions